pkgs/development/compilers/llvm/7/llvm.nix

```diff
@@ -10,15 +10,17 @@
 , version
 , release_version
 , zlib
+, buildPackages
 , debugVersion ? false
 , enableManpages ? false
 , enableSharedLibraries ? true
-, targets ? [ stdenv.hostPlatform stdenv.targetPlatform ]
```

Comment:
@Ericson2314 changed the name as it might be ambiguous when somebody adds a `targets` package.